justfile

```diff
@@ -67,14 +67,31 @@ check-wasm:
 [no-exit-message]
 typos:
   #!/usr/bin/env bash
-  >&2 echo '💡 Valid new words can be added to `_typos.toml`'
-  typos
+  set -eo pipefail
+
+  git_ls_files="$(git ls-files)"
+  git_ls_nonbinary_files="$(echo "$git_ls_files" | xargs file --mime | grep -v "; charset=binary" | cut -d: -f1)"
+
+  if ! echo "$git_ls_nonbinary_files" | parallel typos {} ; then
+    >&2 echo "Typos found: Valid new words can be added to '_typos.toml'"
+    # TODO: not enforcing anything right now, just being annoying in the CLI
+    # return 1
+  fi
```

Comment:
Am I missing something or is this identical to the `check_typos` function?

Reply:
The functionality is the same, but I don't think it's worth to complicate both trying to merge a handful of lines. And if we are going to run it in commit check every time ... then maybe we'll delete this copy soon anyway.